src/App.tsx

```diff
@@ -1,25 +1,35 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getAllTodos } from './api/api';
 
 const App: React.FC = () => {
-  const [
-    selectedUserId,
-    // setSelectedUserId,
-  ] = useState(0);
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [selectedUserId, setSelectedUserId] = useState<number>(0);
+
+  useEffect(() => {
+    getAllTodos()
+      .then(result => setTodosFromServer(result));
```

Comment:
you should handle errors here, if getAllTodos function return an error you have to catch this error here